### PR TITLE
[Open-Sora-Plan-v1.2.0]: VAE training update

### PR DESCRIPTION
**Pull request by wtomin**
Support training causalvae with full precision while using recompute. 

The training step time is 5.45s. The SSIM and PSNR scores are 0.88 and 29.28. The loss is converging normally for over 40k steps.

### CODE REVIEW

**Review by Songyuanwei — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

File: examples/opensora_pku/README.md [hunk 1]
```diff
@@ -341,11 +341,11 @@ Runing this command will generate reconstructed videos under the given `output_g
...
-| CausalVAE  |  8     |       1    |  9x256x256  |     3 mins   |   FP32     |    TRUE      | OFF  |    O0   |    2.04   |  35.29 |  29.30 |   0.88    |
-| CausalVAE  |  8    |       1     | 25x256x256  |    3 mins   |   BF16     |     FALSE    |  OFF |     O0  |     4.21   |  47.51 | 28.92 |    0.87    |
-
+| model name  | cards  |  batch size | resolution | graph compile | precision | discriminator |sink | jit level| s/step | img/s  | psnr | ssim  | train steps|
+|:-----------|:------ |:-----------:|:----------:|:-------------:|:----------:|:------------:|:---:|:--------:|--------:|------:|:----:|-------:|-------:|
+| CausalVAE  |  8     |       1    |  9x256x256  |     3 mins   |   FP32     |    TRUE      | OFF  |    O0   |    2.04   |  35.29 |  29.30 |   0.88    | 40k |
+| CausalVAE  |  8    |       1     | 25x256x256  |    3 mins   |   BF16     |     FALSE    |  OFF |     O0  |     4.21   |  47.51 | 28.92 |    0.87    | 24k |
+| CausalVAE  |  8    |      1      |  25x256x256   |   3 mins   | FP32       |     TRUE    |  OFF |     O0  |   5.45 |   36.70  | 29.28 |   0.88   |  40k |
```
Comment (SamitHuang) [inline examples/opensora_pku/README.md:348]: add `recipe` to the last column.

Comment (SamitHuang) [reply examples/opensora_pku/README.md:348]: train steps can be removed.
row for 9x256x256 can be removed

Comment (wtomin) [reply examples/opensora_pku/README.md]: Updated.

File: examples/opensora_pku/README.md [hunk 2]
```diff
@@ -341,11 +341,10 @@ Runing this command will generate reconstructed videos under the given `output_g
 
 Here, we report the training performance and evaluation results on the UCF-101 dataset. Experiments are tested on Ascend 910* with mindspore 2.3.1 graph mode.
 
-| model name  | cards  |  batch size | resolution | graph compile | precision | discriminator |sink | jit level| s/step | img/s  | psnr | ssim  |
-|:-----------|:------ |:-----------:|:----------:|:-------------:|:----------:|:------------:|:---:|:--------:|--------:|------:|:----:|-------:|
-| CausalVAE  |  8     |       1    |  9x256x256  |     3 mins   |   FP32     |    TRUE      | OFF  |    O0   |    2.04   |  35.29 |  29.30 |   0.88    |
-| CausalVAE  |  8    |       1     | 25x256x256  |    3 mins   |   BF16     |     FALSE    |  OFF |     O0  |     4.21   |  47.51 | 28.92 |    0.87    |
-
+| model name  | cards  |  batch size | resolution | graph compile | precision | discriminator |sink | jit level| s/step | img/s  | psnr | ssim  | recipe|
```
Comment (Songyuanwei) [inline examples/opensora_pku/README.md:344]: 加一列recompute？

Comment (SamitHuang) [inline examples/opensora_pku/README.md:344]: put `graph compile` before the column of `s/step`

Comment (wtomin) [reply examples/opensora_pku/README.md]: OK. Updated.

Comment (wtomin) [reply examples/opensora_pku/README.md]: Updated.

**Review by CaitinZhao — APPROVED** [review]
